### PR TITLE
Use twine and upload to artifactory

### PR DESCRIPTION
**Pull request by bnguyen12**
Using `twine` instead of `setuptools` because `setuptools` is causing build problems for the `Solo` repo

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -10,7 +10,7 @@ def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
 
-install_requires_list = []
+install_requires_list = ["twine>=1.7"]
```
Comment (ethorne) [inline setup.py:13]: do we need to remove a reference to distutils anywhere? or was that just installed by default...?

**Review by ethorne — APPROVED** [review]
LGTM!

File: setup.py (hunk 1 above)
Comment (bnguyen12) [reply setup.py:13]: I don't see references to it in this repo so I think it was installed by default

Comment (ethorne) [reply setup.py:13]: I came to the same conclusion. Aight, ship it then! Once you get that build error ironed out

**Pull request merged**